lib/webdriver.py

```diff
@@ -139,8 +139,9 @@ def _get_driver(self) -> Driver:
             driver_version=driver_version,
             headless=True,
             uc_cdp_events=True,
-            undetectable=True,
+            uc=True,
             is_mobile=True,
+            incognito=True,
```

Comment:
This parameter is already being used in #283 to better avoid 403s/429s. Do you think it's necessary to include in this PR or can it be merged with #283?